### ISSUE TITLE
Uptobox failed - exceptions.OSError

### ISSUE THREAD

**Issue by bast69** [pyload/pyload#2275]
Hi,
I have an error when i want to download with uptobox :
<code>Download failed: mymovie.mp4 | 'exceptions.OSError' object has no attribute 'strip'</code>

In log, i have this :
<code>HOSTER UptoboxCom[58]: Checking for link errors...</code>
<code>HOSTER UptoboxCom[58]: No errors found</code>
<code>HOSTER UptoboxCom[58]: Processing as free download...</code>
<code>HOSTER UptoboxCom[58]: Checking for link errors...</code>
<code>HOSTER UptoboxCom[57]: Plugin may be unstable</code>
<code>HOSTER UptoboxCom[58]: No errors found</code>

I downloaded the archive's pyload and put the folder "module" in /usr/share/pyload. And the folder ~/.pyload/userplugins has been deleted.

**Comment by GammaC0de** [comment]
Debug log please

**Comment by youlJunior** [comment]
Isn't it same as #1987 and #1946 ?

Can you check if the user wich is used by Pyload have enough rights/permissions to write in your download folder ?

**Comment by bast69** [comment]
@youlJunior I just have to check, and my download folder is in "chmod 777"
@GammaC0de the debug log :
[debug_UptoboxCom_08-01-2016_17-39-14.zip](https://github.com/pyload/pyload/files/82985/debug_UptoboxCom_08-01-2016_17-39-14.zip)

**Comment by youlJunior** [comment]
In debug log, you can clearly see the original error :+1: 
```
dl_filename = u'/Downloads/Videos/One Piece/[Kaerizaki-Fansub]_One_Piece_721_[VOSTFR][HD_1280x720].MaChO@zone-telechargement.com.mp4'
	                   e = OSError(13, 'Permission denied')
```

It means that Pyload can't write the file /Downloads/Videos/One Piece/[Kaerizaki-Fansub]_One_Piece_721_[VOSTFR][HD_1280x720].MaChO@zone-telechargement.com.mp4

**Comment by GammaC0de** [comment]
Check the `permissions` subsection of the `general` settings of `pyLoad`

**Comment by bast69** [comment]
Thanks for your replies, i have this in my "general" settings : 
![image](https://cloud.githubusercontent.com/assets/6906908/12215832/d164d5f2-b6cc-11e5-9dba-cbf137d45497.png)
And the permissions for my folder "Download" is : 
![image](https://cloud.githubusercontent.com/assets/6906908/12215838/0eb76866-b6cd-11e5-8cfe-7069def9941c.png)


Any suggestions ?

**Comment by youlJunior** [comment]
Your "Video" folder has 775 permission and is owned by pi:4242.

What is the user which runs pyload (ps -ef | grep pyload) ?

**Comment by bast69** [comment]
the result of <code>ps -ef | grep pyload</code>
![image](https://cloud.githubusercontent.com/assets/6906908/12215900/2ce9c7d6-b6d0-11e5-87c9-e7ce561281b5.png)

**Comment by youlJunior** [comment]
Mmm I can't see your pyload process, maybe it does not contains "pyload" inthe command, can you try "ps -ef | grep python" ?

**Comment by bast69** [comment]
![image](https://cloud.githubusercontent.com/assets/6906908/12215928/393f034c-b6d1-11e5-9323-55ea1c2851ee.png)

**Comment by youlJunior** [comment]
I see.

Your download folder is : /home/pi/.pyload/Downloads
But pyload try to write in /Downloads

What is the folder configured in pyload ?

**Comment by Web-Eye** [comment]
try to enter your download folder with absolute path in settings (config|general|general)
eg: "/home/pyload/.pyload/Downloads" instead of "Downloads"

since yesterday pyload has problems with relative download paths

**Comment by Web-Eye** [comment]
After an update,it's working again!
see #2277

**Comment by bast69** [comment]
Ok, that's work fine ! Thanks you all :+1: 
With the absolute path, downloads work now.
However, i don't update with  the commit https://github.com/pyload/pyload/commit/6b7ef24ac13014c765139719d475995d51db69b3

**Comment by Web-Eye** [comment]
sorry, my fault, it does not work after update and with relative paths! I've just noticed that pyload has created a download folder in root directory, instead of the right download directory....

just for your Information:

pyload home directory: /home/pyload/.pyload
Download directory: /home/pyload/.pyload/Downloads
Download Directory in settings: "Downloads"

This worked till yesterday, now it's a little strange, in Webinterface pyload shows the right folder (/home/pyload/.pyload/Downloads) But, if I download something, pyload want to save it in "/Downloads" And this works only, if I got root rights, otherwise the OSError appears.

So, If I enter the whole absolute path in pyload settings, all works fine. sorry for the too fast message...
